pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -55,6 +55,7 @@
 import org.apache.avro.io.BinaryEncoder;
 import org.apache.avro.io.EncoderFactory;
 import org.apache.avro.util.Utf8;
+import org.apache.hadoop.util.StringUtils;
```

Comment:
We don't tend to use this library

Reply:
My intent was to use `org.apache.commons.lang3.StringUtils;` but imported the wrong one :(